### PR TITLE
adding xml shitty parsing script

### PR DESCRIPTION
**Pull request by jmi4**

### CODE REVIEW

File: read_stig_xml.rb [hunk 1]
```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/ruby
```
Comment (pburkholder) [inline read_stig_xml.rb]: How about:

```
#!/usr/bin/env ruby
```

**Comment by pburkholder** [comment]
Works for me with the env change. Nice work!

**Comment by jmi4** [comment]
Updated the env

File: read_stig_xml.rb (hunk 1 above)
Comment (paulczar) [reply read_stig_xml.rb]: yeah this would be a good idea.

File: read_stig_xml.rb [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+require 'nokogiri'
```
Comment (paulczar) [inline read_stig_xml.rb:2]: yay nokogiri!    I guess we should add a Gemfile to the repo ?

Comment (jmi4) [reply read_stig_xml.rb:2]: I can do that with this PR, should I also require optparse in the Gemfile?

**Pull request merged**